ffi/src/engine_funcs.rs

```diff
@@ -54,6 +54,8 @@ impl Drop for FileReadResultIterator {
 ///
 /// The iterator must be valid (returned by [`read_parquet_file`]) and not yet freed by
 /// [`free_read_result_iter`]. The visitor function pointer must be non-null.
+///
+/// [`free_engine_data`]: crate::free_engine_data
```

Comment:
what's this?

Reply:
ohh nm, i guess it creates the link in the docs?

Reply:
yep!